routers/imagesRouter.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+const express = require('express')
+const MongoClient = require('mongodb').MongoClient
+const MsLogger = require('@first-lego-league/ms-logger').Logger()
+const {authroizationMiddlware} = require('@first-lego-league/ms-auth')
+
+const {getAllImages, getImage, saveImageFromBase64, deleteImage} = require('../logic/images')
+
+const adminAction = authroizationMiddlware(['admin', 'development'])
+
+const router = express.Router()
+
+router.get('/all', (req, res) => {
+  res.send(getAllImages())
+})
+
+router.get('/:imageName', (req, res) => {
+  res.send(getImage(req.params.imageName))
```

Comment:
If you throw an error for an image that doesn't exist, you sold catch it

Reply:
Not here, beacuse its causes 500 and sends the error to the user. Express handles it for me

Reply:
It's kind of a bad practice... The client should get a huma-readable error